scripts/lxd-images

```diff
@@ -91,9 +91,18 @@ class UnixHTTPConnection(http.client.HTTPConnection):
 
 
 class LXD(object):
+    workdir = None
+
     def __init__(self, path):
         self.lxd = UnixHTTPConnection(path)
 
+        # Create our workdir
+        self.workdir = tempfile.mkdtemp()
+
+    def __del__(self):
```

Comment:
Yeah, not sure how much we care about this right now with lxd-images going away real soon.

It looks like I've been lucky though as I've yet to see a case where it doesn't trigger :)